tests/Microsoft.DotNet.Framework.Docker.Tests/DockerHelper.cs

```diff
@@ -42,7 +43,8 @@ public DockerHelper(ITestOutputHelper outputHelper)
             string targetArg = target == null ? string.Empty : $" --target {target}";
             string dockerfileArg = dockerfile == null ? string.Empty : $" -f {dockerfile}";
 
-            ExecuteWithLogging($"build -t {tag}{targetArg}{buildArgsOption}{dockerfileArg} {contextDir}");
+            var stdOut = ExecuteWithLogging($"build -t {tag}{targetArg}{buildArgsOption}{dockerfileArg} {contextDir}");
+            Assert.DoesNotContain(stdOut, "StatusPending");
```

Comment:
There is already an existing test for validating the ngen queue.  See the changes in https://github.com/microsoft/dotnet-framework-docker/pull/463.  Please remove this validation check here as well as the `ngen.exe display` calls in the Dockerfiles in favor of using the existing test.  Currently the test is skipping execution for the versions that were causing failures.  You'll want to enable testing there by removing the conditional logic at https://github.com/microsoft/dotnet-framework-docker/blob/29030b46e5b6fecb0a255d4436878269b168c711/tests/Microsoft.DotNet.Framework.Docker.Tests/AspnetImageTests.cs#L61 and https://github.com/microsoft/dotnet-framework-docker/blob/29030b46e5b6fecb0a255d4436878269b168c711/tests/Microsoft.DotNet.Framework.Docker.Tests/WcfImageTests.cs#L59

Reply:
Thanks for the checking. I removed the validation check code block.

Reply:
@HarveyLink - Sorry for not being clear before but you'll need to enable ngen testing on these Dockerfiles you've changed by following the instructions on the two comments in the test files that I referenced in my previous comment.  Currently ngen is being skipped for these Dockerfiles because they'd end up failing.  Now that you've fixed them, the tests can be enabled.

Reply:
Got it, i have enabled the test, please review.